inst/include/interface/rcpp/rcpp_interface.hpp

```diff
@@ -632,25 +635,31 @@ RCPP_MODULE(fims) {
 
   Rcpp::class_<FleetInterface>("Fleet")
       .constructor()
-      .field("is_survey", &FleetInterface::is_survey)
       .field("log_q", &FleetInterface::log_q)
       .field("log_Fmort", &FleetInterface::log_Fmort)
       .field("nages", &FleetInterface::nages)
       .field("nyears", &FleetInterface::nyears)
       .field("nlengths", &FleetInterface::nlengths)
       .field("estimate_q", &FleetInterface::estimate_q)
       .field("random_q", &FleetInterface::random_q)
-      .field("log_expected_index", &FleetInterface::log_expected_index)
-      .field("proportion_catch_numbers_at_age", &FleetInterface::proportion_catch_numbers_at_age)
-      .field("proportion_catch_numbers_at_length", &FleetInterface::proportion_catch_numbers_at_length)
-      .field("age_length_conversion_matrix", &FleetInterface::age_length_conversion_matrix)
-      .method("SetObservedAgeCompData", &FleetInterface::SetObservedAgeCompData)
+      .field("observed_landings_in_weight", &FleetInterface::observed_landings_in_weight)
+      .field("observed_index_in_weight", &FleetInterface::observed_index_in_weight)
+      .field("log_index_expected", &FleetInterface::log_index_expected)
+      .field("log_landings_expected", &FleetInterface::log_landings_expected)
+      .field("composition_numbers_at_age", &FleetInterface::composition_numbers_at_age)
+      .field("composition_numbers_at_length", &FleetInterface::composition_numbers_at_length)
```

Comment:
what is difference between composition and proportion?

Reply:
Composition is the raw numbers, currently NAA*selectivity, while proportion is scaled to sum to one. There is commented out code in the fleet calcs that tries to rescale composition to match the input data but I took it out while debugging and haven't tried putting it back yet. Comparing observed to expected comps was really hard previously because the expected values were in proportion and raw were in total landings not in comp units.